styles.css

```diff
@@ -0,0 +1,44 @@
+body {
```

Comment:
You must be write to be responsive because the background image might not work well on smaller screens.